binary/grakn.bat

```diff
@@ -54,7 +54,7 @@ goto exiterror
 :startconsole
 
 set "G_CP=%GRAKN_HOME%\console\conf\;%GRAKN_HOME%\console\services\lib\*"
-if exist .\console\services\lib\io-grakn-console-grakn-console-*.jar (
```

Comment:
can we also do the same change in `binary/grakn` script? i think this is much cleaner

Reply:
@lolski addressed in https://github.com/graknlabs/common/pull/81/commits/19e19e0f8eb8695166a6dbb0c3a1cc1d2e463870